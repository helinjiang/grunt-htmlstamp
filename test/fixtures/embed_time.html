<link rel="stylesheet" href="./test1.css">
<script type="text/javascript" src="./test1.js"></script>
<script type="text/javascript" src="./test2.js"></script>
<script type="text/javascript"></script>