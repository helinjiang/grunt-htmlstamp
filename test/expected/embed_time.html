<link rel="stylesheet" href="./test1.151106170914.css">
<script type="text/javascript" src="./test1.151106170914.js"></script>
<script type="text/javascript" src="./test2.js"></script>
<script type="text/javascript"></script>